.gitignore

```diff
@@ -11,6 +15,10 @@ target/
 # Flyway database migration credentials file.
 sqlstore/flyway.properties
 
+# Generated migrations
+sqlstore/src/main/resources/db/migration/afterMigrate.sql
+sqlstore/src/main/resources/db/migration/beforeMigrate.sql
```

Comment:
I didn't notice this before. Shouldn't they generate into `target` rather than `src`?